mount-partitions.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# Mounts boot, state, & data partitions from balenaOS.
+# The container must be privileged for this to function correctly.
+
+use_legacy_message="Using legacy host mounts."
+
+# Get the current boot device in case there are duplicate partition labels
+# for `(balena|resin)-(boot|state|data)` found. The partition identifer `pN`
+# is removed from the end so we can compare with the base device.
+current_boot_device=$(fdisk -l | grep '*  ' | cut -d "p" -f1 2>&1)
+if [ "${current_boot_device}" = "" ] && [ "${TEST}" != 1 ]; then
+    echo "WARNING: Could not determine boot device. Please launch Supervisor as a privileged container with host networking."
```

Comment:
this should probably be prefixed by `ERROR` since it terminates execution

Reply:
Added this [here](https://github.com/balena-os/balena-supervisor/pull/2120/files#diff-8a96bc9af661488f86b92785e9382315f61c548974c4bcce92c1a1deb9a129f8R13)